packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -2321,6 +2327,9 @@ class _WidgetInspectorState extends State<WidgetInspector>
 
 /// Mutable selection state of the inspector.
 class InspectorSelection {
+  /// Root render object in the inspector widget tree.
```

Comment:
this object isn't really related to the selection so it is a bit confusing to have as a public member of the InspectorSelection object. Can you remove it from here and maybe only track it in _RenderInspectorOverlay? That would seem like a great place to be tracking the root render object that we are trying to render relative to.